.babelrc

```diff
@@ -1,3 +1,9 @@
 {
-  "stage": 1
-}
\ No newline at end of file
+  "plugins": [
+    ["transform-runtime", { "polyfill": true, "regenerator": true }]
+  ],
+  "presets": [
+    "es2015",
+    "stage-0"
+  ]
```

Comment:
I didn't look too deep, but doesn't current babel-compile only support a single preset? Also it doesn't support babel plugins.